src/org/usfirst/frc4904/robot/humaninterface/HumanInterfaceConfig.java

```diff
@@ -0,0 +1,12 @@
+package org.usfirst.frc4904.robot.humaninterface;
+
+import org.usfirst.frc4904.robot.vision.AligningCameraGRIP;
+import org.usfirst.frc4904.sovereignty.strategies.GearAlign;
+
+public class HumanInterfaceConfig {
```

Comment:
Not sure about the name (not really a "Config" but can't come up with a better one - guess it's fine since it's yearly code